SIPS/sip-futures_reference_price_methodology.md

```diff
@@ -0,0 +1,63 @@
+---
+sip: 59
```

Comment:
Better to not name it yet as we have another PR with SIP-59 (delegated migrator)

Reply:
Changed to 62.